pytorch_lightning/trainer/trainer.py

```diff
@@ -118,6 +121,7 @@ def __init__(
             profiler: Optional[BaseProfiler] = None,
             benchmark: bool = False,
             reload_dataloaders_every_epoch: bool = False,
+            auto_lr_find: bool = False,
```

Comment:
should this type hint be `Union[bool, str]`?

Reply:
yes it should, will update